runner/runner.go

```diff
@@ -32,7 +34,7 @@ const (
 )
 
 func (p ProcessState) IsDone() bool {
-	return p == UNKNOWN || p == COMPLETE || p == FAILED || p == ABORTED || p == TIMEDOUT
+	return p == COMPLETE || p == FAILED || p == ABORTED || p == TIMEDOUT
```

Comment:
I thought UNKNOWN should be here since it seems natural to check for errors after we're done. What case are you handling by removing it?